exporters/prometheus/testdata/counter.txt

```diff
@@ -1,3 +1,6 @@
 # HELP foo a simple counter
 # TYPE foo counter
-foo{A="B",C="D",E="true",F="42"} 24.3
+foo{A="B",C="D",E="true",F="42",instance="dev",job="test/prometheus_exporter"} 24.3
+# HELP target_info Target metadata
+# TYPE target_info gauge
```

Comment:
For target_info, please test a case with a non-service resource attribute (e.g. telemetry.sdk.version)

Reply:
Changed the test to use the default `Resource` (which includes non-service attributes) and also added another one with custom attributes.